html/Exercise 1/index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Exercise 1</title>
+    <style>
+
+        body {
+            background-color: deepskyblue;
```

Comment:
Don't use such colors, these might not be supported consistently across browsers. Use RGB or hex (hash) nottation